azurerm/internal/services/powerbidedicated/tests/resource_arm_powerbi_dedicated_capacity_test.go

```diff
@@ -0,0 +1,213 @@
+			{
+				Config: testAccAzureRMPowerBIDedicatedCapacity_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMPowerBIDedicatedCapacityExists(data.ResourceName),
+				),
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
+func TestAccAzureRMPowerBIDedicatedCapacity_requiresImport(t *testing.T) {
+	if !features.ShouldResourcesBeImported() {
+		t.Skip("Skipping since resources aren't required to be imported")
+		return
+	}
+
+	data := acceptance.BuildTestData(t, "azurerm_powerbi_dedicated_capacity", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMPowerBIDedicatedCapacityDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMPowerBIDedicatedCapacity_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMPowerBIDedicatedCapacityExists(data.ResourceName),
+				),
+			},
+			{
+				Config:      testAccAzureRMPowerBIDedicatedCapacity_requiresImport(data),
+				ExpectError: acceptance.RequiresImportError("azurerm_powerbi_dedicated_capacity"),
+			},
+		},
+	})
+}
+
+func TestAccAzureRMPowerBIDedicatedCapacity_complete(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_powerbi_dedicated_capacity", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMPowerBIDedicatedCapacityDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMPowerBIDedicatedCapacity_complete(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMPowerBIDedicatedCapacityExists(data.ResourceName),
+					resource.TestCheckResourceAttr(data.ResourceName, "sku_name", "A2"),
+					resource.TestCheckResourceAttr(data.ResourceName, "tags.ENV", "Test"),
+				),
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
+func TestAccAzureRMPowerBIDedicatedCapacity_update(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_powerbi_dedicated_capacity", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMPowerBIDedicatedCapacityDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMPowerBIDedicatedCapacity_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMPowerBIDedicatedCapacityExists(data.ResourceName),
+					resource.TestCheckResourceAttr(data.ResourceName, "sku_name", "A1"),
+				),
+			},
+			data.ImportStep(),
+			{
+				Config: testAccAzureRMPowerBIDedicatedCapacity_complete(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMPowerBIDedicatedCapacityExists(data.ResourceName),
+					resource.TestCheckResourceAttr(data.ResourceName, "sku_name", "A2"),
+				),
+			},
+			data.ImportStep(),
+		},
```

Comment:
could we go back to basic here & do another import step?

Reply:
Updated